main.go

```diff
@@ -216,6 +216,7 @@ func decode(ctx context.Context, method int, protoData *ProtoData) {
 	processed := false
 	start := time.Now()
 	result := ""
+        ar_quests := config.Config.Ar_Quests
```

Comment:
This shouldn't happen here. The presence of have_ar in the header should override the config, so you should only apply this if have_ar is nil in decodeQuest